modules/dashboard/help/dashboard.md

```diff
@@ -2,39 +2,32 @@
 
 Welcome to the LORIS database.
 
-The top menu bar provides easy access to LORIS' many modules and features.
+Access the different modules within LORIS using the top menu bar.
 
-Click on the question mark icon in the top menu to access detailed help information for the current module or page. This Help section provides context sensitive information on navigating and interacting with features within LORIS.
+Every module has a summary page, similar to this one, accessed by the question mark icon in the top right corner. This is designed to give you an overview of the module, what you can do with it, and how.
 
-Upon logging into LORIS, users will see the dashboard. To return to the main dashboard at any time, click on "LORIS" at the far left corner of the menu bar.
-
-Key user information is displayed at the right side of the Menu Bar, including the user's name and the sites to which the user belongs.
-
-To log out from LORIS, click on the user's name and select "Log Out" from the dropdown menu, or select "User Preferences" to update user information, profile settings, and to change your password.
-
-Click on any of the six main menus to access LORIS modules.
+Need to update your account information, such as your password or other profile settings? Click on your name in the top right corner and click **User Preferences**. 
 
 ## Dashboard Features
 
-The dashboard is composed of panels showing at-a-glance status of the LORIS study. It has panels for each of: Recruitment, Study Progression, My Tasks, Document Repository Notifications, and Behavioral Feedback Notifications. Each panel can be hidden by clicking the arrow located in the top right corner.
+This is where you’ll find an at-a-glance status of your LORIS study. Note that all data shown in these sections are from ineligible, excluded, and consent-withdrawn candidates. You can temporarily hide any of the sections from view by clicking the arrow located in the top right corner. We’ve broken down the different sections below:
```

Comment:
> Note that all data shown in these sections are from ineligible, excluded, and consent-withdrawn candidates.

That doesn't sound right..

Reply:
What would be better?

Reply:
Having the dashboard show data from valid candidates instead of ineligible/excluded/consent-withdrawn ones?

Reply:
Is that what it does or is that what it should do? Genuinely asking.

We should have the documentation reflect the current behaviour.

Reply:
Hopefully it's what it does.. if the dashboard is only showing the recruitment graphs and numbers for invalid candidates, it's a pretty serious bug with the dashboard.

Reply:
@knoll-alex The original text says

> Note that charts include data from ineligible, excluded, and consent withdrawn candidates

Which is quite different from saying that ALL the data are from these candidates. 

Could you explain the change? @christinerogers maybe you can weigh in here if this was a collaborative edit.

Reply:
totally my bad - must have misunderstood the original wording but this makes much more sense. Just edited!